app/src/main/java/com/kickstarter/models/PostCommentInput.kt

```diff
@@ -0,0 +1,44 @@
+package com.kickstarter.models
```

Comment:
Move this model into `com.kickstarter.services.mutations` you can see in there most of the data models used on graphql mutations.

-also a small note,  no need for the model that goes in a mutation to be parcelable. We send information, so need to share that model on an Intent.

Reply:
done